app/controllers/refinery/teams/admin/teams_controller.rb

```diff
@@ -4,39 +4,44 @@ module Admin
       class TeamsController < ::Refinery::AdminController
 
         crudify :'refinery/teams/team',
-                :order => 'position ASC',
-                :include => [:translations],
-                :paging => false
+                order: 'position ASC',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.